<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>720</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>-1</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>RSA</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: url(:/bg/bg.jpg) center left; font-size: 13px; color: white</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>741</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #080D2B;border-radius: 20px</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTextBrowser" name="outputDesifrovat">
     <property name="geometry">
      <rect>
       <x>375</x>
       <y>10</y>
       <width>351</width>
       <height>201</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid white; border-radius: 10px</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Zadejte text:</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="inputDesifrovat">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>271</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid white; border-radius: 10px</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>90</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>→</string>
     </property>
    </widget>
    <widget class="QPushButton" name="desifrovat">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>170</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #353686; border-radius: 20px;</string>
     </property>
     <property name="text">
      <string>Dešifrovat</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>741</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #080D2B;border-radius: 20px</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTextBrowser" name="outputZasifrovat">
     <property name="geometry">
      <rect>
       <x>375</x>
       <y>10</y>
       <width>351</width>
       <height>201</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid white; border-radius: 10px</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWrapMode">
      <enum>QTextEdit::WidgetWidth</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Zadejte text:</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="inputZasifrovat">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>271</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid white; border-radius: 10px</string>
     </property>
    </widget>
    <widget class="QPushButton" name="zasifrovat">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>169</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #353686; border-radius: 20px;</string>
     </property>
     <property name="text">
      <string>Zašifrovat</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>80</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>→</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>611</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #080D2B;border-radius: 20px</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>n:</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="keyN">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>561</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid white; border-radius: 10px</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="keyE">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>561</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid white; border-radius: 10px</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>e:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>d:</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="keyD">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>561</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid white; border-radius: 10px</string>
     </property>
    </widget>
    <widget class="QPushButton" name="generovatKlice">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #353686; border-radius: 20px;</string>
     </property>
     <property name="text">
      <string>Generovat klíče</string>
     </property>
    </widget>
    <widget class="QPushButton" name="potvrditKlice">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>160</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #353686; border-radius: 20px; color: white</string>
     </property>
     <property name="text">
      <string>Potvrdit vlastní klíče</string>
     </property>
    </widget>
    <zorder>label_4</zorder>
    <zorder>keyN</zorder>
    <zorder>label_5</zorder>
    <zorder>label_6</zorder>
    <zorder>keyD</zorder>
    <zorder>generovatKlice</zorder>
    <zorder>potvrditKlice</zorder>
    <zorder>keyE</zorder>
   </widget>
   <widget class="QTextBrowser" name="error">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>30</y>
      <width>151</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #080D2B;border-radius: 20px</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="assets/source.qrc"/>
 </resources>
 <connections/>
</ui>
